### PR TITLE
FHIR Stu3 helpers update according to new CodeSystems

### PR DESCRIPTION
**Pull request by salehgamil**
In this PR the FHIR Stu3 helpers were updated to match the new CodeSystems

Jira Ticket -> [https://gesundheitscloud.atlassian.net/browse/SFHXF-271](url)

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+      </option>
+      <option name="PACKAGES_IMPORT_LAYOUT">
+        <value>
+          <package name="" alias="false" withSubpackages="true" />
+          <package name="java" alias="false" withSubpackages="true" />
+          <package name="javax" alias="false" withSubpackages="true" />
+          <package name="kotlin" alias="false" withSubpackages="true" />
+          <package name="" alias="true" withSubpackages="true" />
+        </value>
+      </option>
```
Comment (sebastianwoinar) [inline .idea/codeStyles/Project.xml:19]: Are these changes really needed?

Comment (salehgamil) [reply .idea/codeStyles/Project.xml:19]: I am not sure, I took a look in the .gitignore file and the code styles are not ignored, in fact they're included. They were created by Intellij, I could take them out.

File: fhir-helper/src/androidMain/kotlin/care/data4life/sdk/helpers/DocumentReferenceBuilder.kt [hunk 2]
```diff
@@ -98,7 +98,7 @@ object DocumentReferenceBuilder {
         }
 
         val content = attachments.map { DocumentReference.DocumentReferenceContent(it) }
-        val document = DocumentReference(content, indexed, status, type)
+        val document = DocumentReference(status, type, indexed, content)
```
Comment (wmontwe) [inline fhir-helper/src/androidMain/kotlin/care/data4life/sdk/helpers/DocumentReferenceBuilder.kt:101]: It also changed the order into the FHIR desired one instead of alphabetical -> need to add this to the migration guide

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (wmontwe) [reply .idea/codeStyles/Project.xml:19]: This is added when you import the project for the first time, it is actually correct. When you develop and upgrade the IDE this change should happen, but sometimes it's not triggered until you remove it and reimport the project again. All good here.

**Review by wmontwe — APPROVED** [review]
:+1:

Discovered that we forgot to mention the changed property order in the FHIR release

**Review by wmontwe — CHANGES_REQUESTED** [review]
Could you please change the branch name to: `feature/SFHXF-271/change-to-new-fhir-library-1.0.0`

Including the ticket number in the branch name allows back tracking of work status within Jira.

and add an entry to the CHANGELOG.adoc file.

Thank you!

**Comment by salehgamil** [comment]
@wmontwe  I wasn't able to delete the remote branch, I could push a new branch with the name that you requested and with the same changes.

**Comment by salehgamil** [comment]
Create a new PR #4 with a different name hence this PR is obsolete.

**Comment by wmontwe** [comment]
> @wmontwe I wasn't able to delete the remote branch, I could push a new branch with the name that you requested and with the same changes.

@salehgamil this is a currently known limitation, will be fixed soon with updated branch protection rules
